iothub/service/src/DigitalTwin/Models/DigitalTwinGetResponse.cs

```diff
@@ -17,7 +19,7 @@ public class DigitalTwinGetResponse<T>
         internal DigitalTwinGetResponse(T digitalTwin, string eTag = default)
         {
             DigitalTwin = digitalTwin;
-            ETag = eTag;
+            ETag = new ETag(eTag);
```

Comment:
Should the ctor be modified to take an ETag instead of a string?

Reply:
Updated ctor accordingly.